guide/matrix-vs-discord.md

```diff
@@ -35,16 +35,18 @@ I'm sure you can find a lot of valid criticisms of Discord, like [here](https://
-* **Hostile stance against unofficial clients or client modifications**, thus preventing users from opting out of certain annoyances or tracking.
-* **Arbitrary phone number requirements**.
+* **Lack of privacy for private communications**, as private conversations are not only unencrypted, but also actively scanned (scope of which depends on your settings, but some happen even when you disable all the filtering-related toggles). People reported their messages were substituted with cooking recipes when they tried to send something that is considered a virus by Discord.
+* **Excessive tracking some of which cannot be opted-out in a ToS-abiding way**, such as the [science endpoint](https://luna.gitlab.io/discord-unofficial-docs/science.html) and the process logger (for activity status)[^10]. Most third party clients and some client mods do not support science endpoint, nor process detection. To this date, there is no known user banned for not sending the telemetry data.
+* **Hostile stance against unofficial clients or client modifications**, thus preventing users from opting out of certain annoyances or tracking in a compliant way.
+* **Arbitrary phone number requirements**. Some users that are deemed suspicious and users that join certain "servers" are required to verify their phone number.
 * Discord has made **decisions against the users' best interests**, with the most recent ones being:
   * Biased consultations and unilateral proposal of [cryptocurrency integrations](https://www.reddit.com/r/discordapp/comments/qpmhs5/discord_developers_please_do_not_support_nfts/) (which were only dropped after massive backlash); and
-  * Mandatory rollout of slash commands, as well as KYC requirements, for running bots (which were *not* dropped despite [backlash](https://gist.github.com/Rapptz/4a2f62751b9600a31a0d3c78100287f1)).
+  * Deprecating access to read text messages in "servers" with certain exceptions[^12], effectively equating to mandatory rollout of application commands based interaction flow, as well as KYC requirements, for running bots in more than 100 "servers" (which were *not* dropped despite [backlash](https://gist.github.com/Rapptz/4a2f62751b9600a31a0d3c78100287f1)).
 * **Closed source**, thus cannot be independently inspected.
 * **Lack of control for private data and no guarantee on reliability**, as Discord is centralized, not federated. See [here](../matrix-vs-al/#centralized-platforms).
-
+* **Paywalling richer expression abilities**.[^11] A user with Nitro unlocks certain additional features, mainly related to what can be included in a message, and twice as more guild cap, and twice as long message length limit.
+* **Not intended for serious use**. A developer experience staff at Discord explicitly said, "We are just a casual chatting app." Marketing campaigns of Discord, such as inclusion of gaming related figures, and use of funny memes, seem to agree with that stance.
```

Comment:
Discord seems to have moved away from that and tries to appeal to the general population, although it still seems casual. But anyway, this guide mainly addresses people who are either casual or semi-serious (eg. FOSS projects; corporations can look at element.io), so I don't see the point of including that.

Reply:
fair